astropy/units/physical.py

```diff
@@ -7,7 +7,7 @@
 
 from astropy.utils.exceptions import AstropyDeprecationWarning
 
-from . import imperial  # noqa: F401  # Needed for backward namespace compat, see #11975 and #11977
+from . import imperial  # Needed for backward namespace compat, see #11975 and #11977
```

Comment:
@nstarman why is `yesqa` stripping the `# noqa` off this line?

Reply:
Hmm.  maybe because F401 is not in https://github.com/astropy/astropy/blob/3132229f01427ee0637c038c7475cf8b6af6d8ee/.pre-commit-config.yaml#L116

Reply:
Or move the noqa comment to the end

Reply:
maybe similar to https://stackoverflow.com/questions/51179109/set-pyflake-and-mypy-ignore-same-line

Reply:
It's not a fan of having the comment and noqa comment on the same line.

Reply:
Any way that works. 👍